lib/async.js

```diff
@@ -15,12 +15,21 @@ function findAllConfigs(options) {
         });
 
         return Promise.all(realFiles.map(function(file) {
-            return fsHelpers.read(file).then(function(content) {
-                var parsedConfig = nodeEval('(' + content + ')');
+            var parsedConfig;
 
+            try {
+                parsedConfig = require(file);
                 parsedConfig.__source = file;
-                return parsedConfig;
-            });
+
+                return Promise.resolve(parsedConfig);
+            } catch (err) {
+                return fsHelpers.read(file).then(function(content) {
+                    parsedConfig = nodeEval('(' + content + ')');
```

Comment:
I'd not do that, `node-eval` calls `vm.runInThisContext` inside, which is an expensive task so I don't really understand, what profit we get.